processors/src/main/java/com/axellience/vuegwt/processors/component/template/ComponentTemplateProcessor.java

```diff
@@ -82,9 +86,10 @@ public void processComponentTemplate(TypeElement componentTypeElement,
 
         // Parse the template
         TemplateParserResult templateParserResult = new TemplateParser().parseHtmlTemplate(
-            optionalTemplateContent.get(),
+            optionalTemplateContent.get().a,
```

Comment:
Could you create a class (like `TemplateFileResource`) instead of using Pair here? You can make `TemplateParserResult` constructor take it as a parameter. I was wondering about the .a and .b at first ;). Also as Pair is part of javaparser utils and not the language it might be gone at some point